bundles/org.openhab.ui/web/src/pages/settings/things/inbox/inbox-list.vue

```diff
@@ -61,6 +65,7 @@
               By binding
             </f7-button>
           </f7-segmented>
+        <f7-checkbox v-if="showCheckboxes" @change="masterToggle" />
```

Comment:
Not convinced this lone checkbox is the best design for a "select all" feature honestly.

Reply:
I agree, but until now I could not find a better, simple way to provide the functionality.
I tried a header-row, but that needs much more space without additional improvements.

Do you have a suggestion for this?

Reply:
Not completely sure but I was thinking about "Select All/Unselect All" buttons like those you have on that screen when you add points to the model from a thing:

![image](https://user-images.githubusercontent.com/2004147/121579606-2995ff80-ca2c-11eb-9a67-3da3f5f85555.png)

Though your approach of a "select all" checkbox isn't bad either, maybe it could be made clearer, but in any case we need a common approach. So I would suggest adding these buttons at the bottom of the list when the "select" mode is active.

Reply:
What do you think about such a composition?
![image](https://user-images.githubusercontent.com/23188584/121593426-39b5db00-ca3c-11eb-97e3-8ba081228065.png)

This way it would be possible to select all items in bigger lists without scrolling to the bottom.